client/src/index.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
-// import Search from './components/Search.jsx';
+import Search from './components/Search.jsx';
 import ReviewList from './components/ReviewList.jsx';
+import styled from 'styled-components'
```

Comment:
Don't need to import if you aren't using it

Reply:
Removed.